src/utils/style.js

```diff
@@ -144,7 +144,6 @@ export const defaultStyles = {
   'visibility': 'visible',
   'white-space': 'normal',
   'widows': '2',
-  'width': 'auto',
```

Comment:
Add a test that `width`, `left` and `margin-left` never get into this array.

Reply:
Done